package.json

```diff
@@ -14,10 +14,10 @@
     "eslint:check": "eslint . --ignore-path .gitignore --ext .ts,.tsx,.js",
     "prettier": "prettier --ignore-path .gitignore --write \"**/*{.ts,.tsx,.js,.jsx,.css,.json}\"",
     "prettier:check": "prettier --ignore-path .gitignore --check \"**/*{.ts,.tsx,.js,.jsx,.css,.json}\"",
-    "start-jupyter": "mkdir -p build/cypress-tests && jupyter lab --config=./tests/test-config.py",
     "test": "npm run test:unit && npm run test:integration",
-    "test:integration": "start-server-and-test start-jupyter http-get://localhost:58888?token=test cy:run",
-    "test:integration:debug": "start-server-and-test start-jupyter http-get://localhost:58888?token=test cy:open",
+    "start": "mkdir -p build/cypress-tests && ts-node scripts/start-test-server.ts",
+    "test:integration": "server-test 'yarn start' ':9000/minio/health/live|http-get://localhost:58888?token=test' 'yarn cy:run'",
+    "test:integration:debug": "server-test 'yarn start' ':9000/minio/health/live|http-get://localhost:58888?token=test' 'yarn cy:open'",
```

Comment:
What is this minio url being used for?

Reply:
these are the urls for the liveliness checks before starting the cypress tests. Some of the tests depend on minio so I added it's url here too